iree_tests/benchmarks/benchmark_sdxl_rocm.py

```diff
@@ -0,0 +1,63 @@
+import os
+from collections import namedtuple
+
+benchmark_dir = os.path.dirname(os.path.realpath(__file__))
+
+def launchCommand(scriptcommand):
+    print("Launching:", scriptcommand, "[ Proc:", os.getpid(), "]")
```

Comment:
I can't quite tell since this print didn't appear in logs (see my other comment), but I think I'd suggest this syntax be changed. I like to wrap some values in quotes when logging to make it obvious if the command is empty string and where the command stops and starts.

```suggestion
    print(f"Launching: '{scriptcommand}' [Proc: {os.getpid()}]")
```